shared/src/packages/prompts/libs/types/prompt-base-response.dto.type.ts

```diff
@@ -1,9 +1,9 @@
-type GeneratedArticlePrompt = {
+type PromptBaseResponseDto = {
   genre: string;
   character: string;
   setting: string;
   situation: string;
   prop: string;
```

Comment:
On [this comment](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/61/files#r1310602232) I'm talking about this 
```suggestion
  [PromptCategory.GENRE]: string;
  [PromptCategory.CHARACTER]: string;
  [PromptCategory.SETTING]: string;
  [PromptCategory.SITUATION]: string;
  [PromptCategory.PROP]: string;
```